[2023-12-10 19:35:45,533] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-12-14T00:00:00+00:00 [queued]>
[2023-12-10 19:35:45,589] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-12-14T00:00:00+00:00 [queued]>
[2023-12-10 19:35:45,591] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:35:45,599] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:35:45,601] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:35:45,716] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): tercera_tarea> on 2022-12-14 00:00:00+00:00
[2023-12-10 19:35:45,743] {standard_task_runner.py:52} INFO - Started process 1580 to run task
[2023-12-10 19:35:45,782] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'tercera_tarea', 'scheduled__2022-12-14T00:00:00+00:00', '--job-id', '412', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpe3zn554b', '--error-file', '/tmp/tmphubq40f0']
[2023-12-10 19:35:45,807] {standard_task_runner.py:80} INFO - Job 412: Subtask tercera_tarea
[2023-12-10 19:35:47,032] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-12-14T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:35:47,851] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=tercera_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-12-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-14T00:00:00+00:00
[2023-12-10 19:35:47,853] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:35:47,893] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2']
[2023-12-10 19:35:47,952] {subprocess.py:85} INFO - Output:
[2023-12-10 19:35:47,984] {subprocess.py:92} INFO - hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2
[2023-12-10 19:35:47,987] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:35:48,255] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=tercera_tarea, execution_date=20221214T000000, start_date=20231210T193545, end_date=20231210T193548
[2023-12-10 19:35:48,427] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:35:48,889] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
